internal/messaging/messaging_test.go

```diff
@@ -131,6 +132,7 @@ func TestBootstrapHandler(t *testing.T) {
 
 			actual := BootstrapHandler(context.Background(), &sync.WaitGroup{}, startup.NewTimer(1, 1), dic)
 			assert.Equal(t, test.ExpectedResult, actual)
+			assert.Empty(t,test.Config.MessageQueue.Optional)
```

Comment:
there is a fmt issue in messaging_test.go
it looks like you need a space in this line
please make sure `make test` can pass